### ISSUE TITLE
404 page not found

### ISSUE THREAD

**Issue by sabrinakissack** [jmf-nyu/miap_diglit#33]
When you try to open the apt link on the Digital Literacy Command Line Guide page, it takes you to a 404 page not found. I imagine this is because the page has not yet been filled out.

**Comment by jmf-nyu** [comment]
Good catch. Yes, you're correct that it's because the page hasn't been created yet. However, until it's fixed, it is technically a bug in the website.

**Comment by jmf-nyu** [comment]
apt page has been added. This has been fixed.
